pinot-query-planner/src/main/java/org/apache/pinot/query/planner/logical/RexExpression.java

```diff
@@ -31,6 +31,8 @@
  */
 public interface RexExpression {
 
```

Comment:
Are these methods used anywhere ?

Reply:
In fact they are not. Having the visitor for the future may be useful, but it is also very simple to add again, so I'm removing it